tabular/fraud/Models/fraud_model.py

```diff
@@ -7,4 +7,6 @@
 def predict_df(df):
     for col in cat_cols:
         df[col] = df[col].astype(object)
+    if "timestamp" in df:
```

Comment:
would it be slightly less hacky if instead we specified all the features (ie columns that should be there) and only passed those to model? ik it makes no functional diff, but since this is being given to customers to play around with maybe worth making pretty